crm/types.go

```diff
@@ -1,15 +1,43 @@
 package crm
 
 const (
-	VersionUrl = "crm/v1/"
+	VersionCRMUrl = "crm/v1/"
```

Comment:
when you import Version to another package, it already has crm in the title crm.VersionUrl

Reply:
In general, I agree, but then we need to make changes in the types.go files for newsfeed, task and group. For consistency.